openclaw_hooks/nima-recall-live/lazy_recall_v2.py

```
@@ -0,0 +1,529 @@
+if __name__ == "__main__":
+    if len(sys.argv) < 2:
+        print("Usage: lazy_recall_v2.py <query> [affect_json]", file=sys.stderr)
+        sys.exit(1)
+    
+    query = sys.argv[1]
+    current_affect = None
+    
+    if len(sys.argv) > 2:
+        try:
+            current_affect = json.loads(sys.argv[2])
+        except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)